lib/internal/streams/whatwg.js

```diff
@@ -0,0 +1,4283 @@
+'use strict';
```

Comment:
rename to `web.js` ?

Reply:
It's also a very long file. Could it be split into several ones?

Reply:
Eventually, perhaps. While I'm still working on ensuring that the implementation is correct, keeping everything in a single file is the most convenient. Once I get into refactoring for performance optimization I'll look at splitting things up.